package.json

```diff
@@ -80,6 +90,23 @@
                     "default": null,
                     "description": "Specify options to appended only to the 'run suite' command, for example add options like '--testsuite unit' or '--coverage  --coverage-xml'"
                 },
+                "better-phpunit.coveragePath": {
+                    "type": [
+                        "string",
+                        "null"
+                    ],
+                    "default": null,
+                    "description": "Relative path for the generated coverage files, based on the project folder. Ex: 'test/coverage'"
+                },
+				"better-phpunit.coverageFormat": {
```

Comment:
can you fix the weird spacing here please

Reply:
Fixed at my last commit. Maybe it would be useful to introduce a .editorconfig file.